Makefile

```diff
@@ -98,17 +97,17 @@ install.tools: .install.golint .install.govet .install.gitvalidation
 # golint does not even build for <go1.5
 .install.golint:
 ifeq ($(call ALLOWED_GO_VERSION,1.5,$(HOST_GOLANG_VERSION)),true)
-	go get github.com/golang/lint/golint
+	go get -u github.com/golang/lint/golint
 endif
 
 # go vet is now included in >=go1.5, so no need to get it.
 .install.govet:
-ifeq ($(call ALLOWED_GO_VERSION,1.5,$(HOST_GOLANG_VERSION)),true)
-	go get golang.org/x/tools/cmd/vet
+ifneq ($(call ALLOWED_GO_VERSION,1.5,$(HOST_GOLANG_VERSION)),true)
```

Comment:
On Mon, Aug 29, 2016 at 04:03:25PM -0700, Vincent Batts wrote:
> >  endif
> >  
> >  # go vet is now included in >=go1.5, so no need to get it.
> >  .install.govet:
> > -ifeq ($(call ALLOWED_GO_VERSION,1.5,$(HOST_GOLANG_VERSION)),true)
> > -	go get golang.org/x/tools/cmd/vet
> > +ifneq ($(call ALLOWED_GO_VERSION,1.5,$(HOST_GOLANG_VERSION)),true)
> 
> But we're not testing against 1.4 in Travis.

Travis is not the only Makefile consumer.  If it was, you wouldn't
need this check at all.